resources/qml/delegates/ImageMessage.qml

```diff
@@ -34,9 +34,12 @@ Item {
 		fillMode: Image.PreserveAspectFit
 
 		MouseArea {
+            hoverEnabled: true
 			enabled: model.data.type == MtxEvent.ImageMessage && img.status == Image.Ready
 			anchors.fill: parent
 			onClicked: timelineManager.openImageOverlay(model.data.url, model.data.id)
+            ToolTip.visible: hovered
+            ToolTip.text: model.data.id
```

Comment:
Why do you want to show the event id when hovering over the image?

Reply:
Is that what that does? Notice that I don't really do QML (just picked up a little for this).

Reply:
I'll work on it more :)

Reply:
You were accessing this member: https://github.com/Nheko-Reborn/nheko/blob/master/src/timeline/TimelineModel.cpp#L355

You probably want to show something different. I would also appreciate it, if you tried to do something similar to Lurkkis design, to be somewhat consistent (i.e. not using a tooltip, but a somewhat transparent overlay with image description, filename, maybe size and a maximize button or so).

You probably need to build that to properly test it.

Reply:
OK, I'll see what I can do.